src/test/integration/api_request.py

```diff
@@ -0,0 +1,53 @@
+import json
+import requests
+import urllib2
+import ssl
+from configuration_load import *
+
+
+def good_request():
+    query_params = {'q': 'Oxford'}
+    headers = {'Authorization': get_access_token()}
+    request = requests.get(get_url(), params=query_params, headers=headers)
+    return request.status_code
+
+def unauth_request():
+    query_params = {'q': 'Oxford'}
+    request = requests.get(get_url(), params=query_params)
+    return request.status_code
+    
+def not_found_request():
+    query_params = {'campus': 'Pluto'}
+    headers = {'Authorization': get_access_token()}
+    request = requests.get(get_url(), params=query_params, headers=headers)
+    return request.status_code
```

Comment:
For some of these requests that return error objects, could you assert that the fields that we expect are present?